topicgenerator/src/main/java/io/dockstore/topicgenerator/helper/AIModelInterface.java

```diff
@@ -0,0 +1,8 @@
+package io.dockstore.topicgenerator.helper;
+
+public interface AIModelInterface {
```

Comment:
VERY STRONG PERSONAL OPINION:
I do not like `Interface` appearing at the end of our `interface` names.  It's an extra word that doesn't provide much useful information and makes our code harder to read and understand.  If I had my druthers, `SourceCodeRepoInterface` -> `SourceCodeRepo`, etc etc etc